package.json

```diff
@@ -19,5 +19,11 @@
     "package:storeadmin": "lerna run --scope storeadmin --stream dev",
     "dev": "run-p --print-label package:*",
     "build": "lerna run --parallel build"
+  },
+  "dependencies": {
+    "@nestjs/swagger": "^5.2.1",
+    "@types/uniqid": "^5.3.2",
+    "swagger-ui-express": "^4.3.0",
+    "uniqid": "^5.4.0"
```

Comment:
we won't use this uniqid package. rather use nodejs crypto.randomUUID https://nodejs.org/api/crypto.html#cryptorandomuuidoptions